extra/material.js

```diff
@@ -2457,12 +2457,6 @@ function MaterialTab(tab, ctx) {
         }
         tab.addEventListener('click', function (e) {
             e.preventDefault();
-            var href = tab.href.split('#')[1];
-            var panel = ctx.element_.querySelector('#' + href);
-            ctx.resetTabState_();
-            ctx.resetPanelState_();
-            tab.classList.add(ctx.CssClasses_.ACTIVE_CLASS);
-            panel.classList.add(ctx.CssClasses_.ACTIVE_CLASS);
```

Comment:
Yeah, I was trying to avoid making changes, but noticed that the current diff between the two material.js files removed similar functions that are obsolete in React style development.

As it is, the changes I made will work with the current material.js file, but causes errors in the console if I don't have a tab panel and hash-links configured.

I will do some thinking on the problem :)